pkg/cluster/config/validate_test.go

```diff
@@ -22,6 +22,84 @@ import (
 	"sigs.k8s.io/kind/pkg/util"
 )
 
+func TestClusterValidate(t *testing.T) {
```

Comment:
I'm curious, what job run these tests?

Reply:
nothing! :this-is-fine: :^)

I think build used to? need to fix that

Reply:
added a commit to make the build job do that for now